### ISSUE TITLE
CoFHTweaks crashes in "cofh.tweak.asmhooks.render.RenderGlobal$RenderWorker.run(RenderGlobal.java:644)"

### ISSUE THREAD

**Issue by adudney** [CoFH/Feedback#669]
Versions:
Minecraft 1.7.10
MultiMC 5 0.4.7
Mac OSX 10.10.4
Java 8u45
Forge 10.13.4.1448
CoFHTweaks 0.2.0B2-33

Unfortunately I couldn't find a way to reliably reproduce the crash, nor have I been able to reproduce it in a regular world rather than an amplified one.
I initially caused the crash by:
1. Generating a random world using the amplified terrain type (Seed shown below)
2. While flying around the entire screen turned light blue, then the game crashed. I was able to cause it once more and then it stopped happening.

Seed:
![2015-07-10_16 22 47](https://cloud.githubusercontent.com/assets/246232/8630851/2515d670-2720-11e5-97dd-64c16c280bf2.png)

Crash logs: https://gist.github.com/adudney/05b24f37d9fb313b7e1e

**Comment by Serifina** [comment]
This crash is effectively identical to one I had, as well, with the most recent version of Tweaks, but this wasn't an Amplified world. It's a Garden of Glass world (Botania Skyblock).

When I first loaded the world with Tweaks installed - No crash. Played for several hours, everything was fine. Closed the game.

Loaded the world with Tweaks installed the second time - The visuals would load, then the game would freeze before I could do anything, produce the light blue screen adudney mentioned, then the game would crash out. I tried to load the game three times, each time producing an identical crashlog (http://pastebin.com/iji9Rkzu). 

Disabled Tweaks, and played for a while. The re-enabled Tweaks, and no crash.
